README.md

```diff
@@ -1,2 +1,166 @@
-# React-Capstone
-This React capstone project is about building a mobile web application to check a list of metrics (numeric values) that I will create making use of React and Redux.
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Setup](#setup)
+- [👥 Author](#author)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 React Capstone - CoinGecko <a name="about-project"></a>
+
+This React capstone project is about building a mobile web application to check a list of metrics (numeric values) that I will create making use of React and Redux. It's connected to CoinGecko API to get several data about cryptocurrencies.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.hostinger.com/tutorials/what-is-react">REACT</a></li>
+    <li><a href="https://redux-toolkit.js.org/">REDUX TOOLKIT</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>N/A</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **Follow a correct Gitflow**
+- **Comply with linters**
+- **Use React components**
+- **Use React Router**
+- **Understand the concept of state management**
+- **Use store, actions, and reducers in React**
+- **Setup a Redux store**
+- **Create a slice to save state and reducers**
+- **Connect React and Redux**
+- **Connect an existing API via Redux and use the data to display something in a UI**
+- **Apply styling**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live site](https://coingecko-4xi2.onrender.com/metrics)
+- [Loom video](https://www.loom.com/share/2430429713de4206b56c88c4908b3120)
+
```

Comment:
- As previously mentioned, whenever the live link is clicked, the webpage loads infinitely, but when the back button is clicked, the homepage opens up, I think something is either wrong with the deployment or you are using the details page as your root route. Consider looking into this to provide a better UX.

[screen-capture.webm](https://github.com/CarlosIgreda/React-Capstone/assets/84930355/91d3a333-5ed2-4996-abab-c14cc980c328)

Reply:
@DuaneDave done it. The error was on my link "https://coingecko-4xi2.onrender.com/metrics" I fixed it by removing the "/metrics". Now the link is: "https://coingecko-4xi2.onrender.com"